### DISCUSSION TITLE
WinSCP to Zigbee2mqtt - PVE LXC Container

### DISCUSSION THREAD

**Discussion by wildchai** [tteck/Proxmox#1800] [Help Needed]
Hi there,

I want to edit some files in the Z2M LXC container in Proxmox via WinSCP. I can do it for PVE but having trouble getting connected to Z2M container. Both my Proxmox and Z2M were installed using the scripts by tteck.

I have been going through many guides and topics with the same issue as mine and tried their method. I still couldn't get access to my LXC container. Keeps getting an 'Access Denied' error.

I have edited the sshd_config file in both the LXC container and the PVE:

Permitrootlogin yes

PasswordAuthentication yes

Tried to access with root and created a new user in both PAM and PVE still does not work.

Appreciate any help. Thanks.

**Comment by tteck** [comment]
Are you using automatic login for the Z2M container?

**Reply by wildchai** [reply]
Sorry. Can you elaborate on this? Very new to this. Where can I check this? 
I can directly access the Z2M container's console without login... Yes.

**Reply by tteck** [reply]
SSH/WinSCP with autologin is not an option in this case because there are no root credentials available.
